monai/apps/datasets.py

```diff
@@ -45,7 +45,7 @@ class MedNISTDataset(Randomizable, CacheDataset):
 
     Raises:
         ValueError: root_dir must be a directory.
-        RuntimeError: can not find dataset directory, please use download=True to download it.
+        RuntimeError: can not find dataset directory: {dataset_dir}, please use download=True to download it.
```

Comment:
May I know why to use the "{}" here? Can we see some beautiful web pages in our docs?
Thanks.

Reply:
For the Raises I copied verbatim the error messages most of which use f strings hence the "{}". This particular error message must have been updated to tell the direction which it could not find.

![docs](https://user-images.githubusercontent.com/42684402/87858927-1bf8f800-c8f7-11ea-9747-7536cd6c6483.png)

Here's what it looks like. I thought it would be best to copy the exact message but we could change the approach.

Reply:
I would remove the {} and use a syntax for referring to members, I think that's ``. For the raises description this should state what the actual condition is that raises the exception, not the helper message that goes with the exception itself. For ValueError I'd put "When `root_dir` is not a directory." and for RuntimeError "When `dataset_dir` doesn't exist and downloading is not selected (`download` = False)." If there's something more to say about what the condition under which the exception is raise which isn't obvious that would be placed here to. 

For the actual exception strings they should all be correct and complete sentences, so beginning with a capital letter (and so not an identifier which can't be capitalized) and ending with a period. For example: "Cannot find data set directory `dataset_dir`; please use `download` = True."